### ISSUE TITLE
Transparent background of PNG previews gets black

### ISSUE THREAD

**Issue by tobiasfrust** [zenodo/zenodo#1456]
### Problem:
With the recent introduction of the IIIF previewer, PNG images with a transparent background are displayed with black background color.

Example: https://zenodo.org/record/818277 (both png files)

### Screenshots:

Current behavior:
![zenodo_818277](https://user-images.githubusercontent.com/10207535/38557007-ea42efb4-3ccb-11e8-884c-fd5ff15659c2.png)

Expected behavior:
![zenodo_818277_good](https://user-images.githubusercontent.com/10207535/38557009-ece39f48-3ccb-11e8-92b7-7e8ab4170ded.png)

### Possible solution:

The problem disappears, if the original file extensions would be kept in the `invenio-iiif` previewer. Currently, everything is converted to `jpg`. (see https://github.com/inveniosoftware/invenio-iiif/blob/26c1d662e1724c8d2ec6f8f0a80149941bbbca81/invenio_iiif/previewer.py#L34-L42)

Probably, something like this would help:

```python
    extension = os.path.splitext(file.filename)[1][1:]
    return render_template(
        current_app.config['IIIF_PREVIEW_TEMPLATE'],
        file=file,
        file_url=ui_iiif_image_url(
            file.file,
            image_format=extension,
            **current_app.config['IIIF_PREVIEWER_PARAMS']
        )
    )
```

**Comment by lnielsen** [comment]
Thanks for reporting. We'll fix it in Invenio-IIIF

**Comment by lnielsen** [comment]
Fixed in inveniosoftware/invenio-iiif#5
